AutoCollection/HttpRequests.ts

```diff
@@ -282,6 +282,8 @@ class AutoCollectHttpRequests {
          AutoCollectHttpRequests.INSTANCE = null;
          this.enable(false);
          this._isInitialized = false;
+         CorrelationContextManager.disable();
```

Comment:
I'm curious. What does this change do?

Reply:
the call to `disable` isn't necessary, but I needed to add `this._isAutoCorrelation = false` in order to be able to load one of the cls libraries in tests. Since I was setting isAutoCorrelation to false, it just made sense to actually disable auto correlation here. 👍